webapp/src/views/projects/translations/history/History.tsx

```diff
@@ -119,8 +130,22 @@ export const History: React.FC<Props> = ({ translation }) => {
     scrollerRef.current?.scrollTo({ top: scrollerRef.current.scrollHeight });
   }, [history.data?.pages?.[0].page?.totalElements]);
 
+  const [differences, setDifferences] = useState(true);
```

Comment:
This should be named like "showDifferences", like this it's confusing, because it feels like it's actually holds the differences and not the boolean.

Reply:
👍